diff --git a/sample/enum_net.cpp b/sample/enum_net.cpp
index 9111f9f..a9e79b3 100644
--- a/sample/enum_net.cpp
+++ b/sample/enum_net.cpp
@@ -35,7 +35,7 @@ int enum_net(std::vector<std::string>&& args)
 
         auto dev = net.get_dev();
         print(dev);
-        
+
         auto icmp = net.get_icmp();
         print(icmp);
 
@@ -66,8 +66,8 @@ int enum_net(std::vector<std::string>&& args)
         auto udplite6 = net.get_udplite6();
         print(udplite6);
 
-        auto unix = net.get_unix();
-        print(unix);
+        auto unix_ = net.get_unix();
+        print(unix_);
 
         auto netlink = net.get_netlink();
         print(netlink);
